base_optional_quick_create/model.py

```diff
@@ -38,16 +39,42 @@ def wrapper(cr, uid, name, context=None):
         return wrapper
 
     def _register_hook(self, cr, ids=None):
```

Comment:
I think it's dangerous to change `_register_hook`'s signature. Better refactor this into its own method and just call it here